grg_pssedata/struct.py

```diff
@@ -59,6 +59,17 @@ def quote_string(s):
     '''adds PSSE single quotes to a string'''
     return '\'{}\''.format(s)
 
+def _psse_str(s):
+    '''format numbers to PSSE style'''
+    if isinstance(s, float):
+        if abs(s) <= 1e-1:
+            return "{:.5e}".format(s)
+        elif 1e-1 < abs(s) < 1e1:
+            return "{:.5f}".format(s)
+        else:
+            return "{:.3f}".format(s)
```

Comment:
I am generally not a fan of eliminating some floating point precision without a strong context to know it won't break the model.  Could you elaborate on what you where thinking here?

Switching between decimal and scientific does seem reasonable to me.

Reply:
I was thinking of formatting it so that it can be readable from the .RAW file in a more similar way to the .RAW file generate with the PSSE.

I agree that it may not be the best idea. We could discard this change.

Reply:
In the interest of getting your fixes merged soon, I propose we change this function to replicate the the previous behavior.  You can just comment out the formatting suggestion and make the function `return str(s)`.  Sound good?

Reply:
Sure, Sounds good to me.